examples/secured-client-with-jwt-auth/secured_client_with_jwt_auth.description

```diff
@@ -0,0 +1,5 @@
+// A secured client with jwt authentication is used to connect to a
```

Comment:
Shall we change 'jet' to 'JWT'? Fix everywhere to maintain consistency.

Reply:
Fixed.